.pre-commit-config.yaml

```
@@ -6,7 +6,7 @@ repos:
     rev: v4.0.1
     hooks:
       - id: trailing-whitespace
-        exclude: "tests/functional/t/trailing_whitespaces.py"
+        exclude: "tests/functional/t/trailing_whitespaces.py|tests(/.*)*/data"
```

Could we target specifically the html for pyreverse's tests here and for prettier, please ?
```suggestion
        exclude: "tests/functional/t/trailing_whitespaces.py|tests/pyreverse/data/*.html"
```

---

Sure